securemail/securemail.php

```diff
@@ -52,7 +52,7 @@ function securemail_uninstall() {
  *
  * @see App
  */
-function securemail_settings(App &$a, &$s){
+function securemail_settings(App $a, &$s){
```

Comment:
Please add a new line before opening brace. I'm asking because it's also missing a space before the brace, might as well going for the right thing.